acceptance/telemetry/dummy-without-server/out.upload.txt

```diff
@@ -0,0 +1,26 @@
+error: Failed to upload telemetry logs: unable to parse response. This is likely a bug in the Databricks SDK for Go or the underlying REST API. Please report this issue with the following debugging information to the SDK issue tracker at https://github.com/databricks/databricks-sdk-go/issues. Request log:
```

Comment:
This PR and specifically this test is blocked on proper mocking of errors. Please ignore this and review the other bits for now.